### ISSUE TITLE
Strange behavior with fill() autocomplete in talk-through recording

### ISSUE THREAD

**Issue by pamelafox** [Khan/live-editor#349]
I just recorded this talk-through:
https://www.khanacademy.org/computer-programming/special-processingjs-functions/5076945701306368

If you watch at around -2:04, you'll see me write fill(mouseX, mouseY, mouseY);
It's all well and good, but then if you pause at any point, you'll see the fill() autocomplete come in, with 255, 0, 0); and it places it at a bad spot, causing the code to be error-full.

**Comment by kevinb7** [comment]
I don't have computer-programming/special-processingjs-functions/5076945701306368 locally, if I update my datastore will that grab it?

**Comment by pamelafox** [comment]
Probably not, since I made it yesterday. But, I was able to replicate it locally with a quick talk-through:
1. Go to /cs/new/pjs
2. Click record
3. Type
draw = function() {
fill(mouseX, mouseY, mouseZ);
ellipse(10, 10, 10, 10);
}
4. Save recording
5. Play it, and jump forward, see same issue
<img width="365" alt="screen shot 2015-07-07 at 9 35 29 am" src="https://cloud.githubusercontent.com/assets/297042/8551456/86d36996-248b-11e5-9596-16850addc9d4.png">
